core/src/main/java/org/apache/carbondata/core/metadata/converter/ThriftWrapperSchemaConverterImpl.java

```diff
@@ -197,6 +197,7 @@
       thriftColumnSchema.setColumnProperties(properties);
     }
     thriftColumnSchema.setAggregate_function(wrapperColumnSchema.getAggFunction());
+    thriftColumnSchema.setTimeSeriesFunction(wrapperColumnSchema.getTimeSeriesFunction());
```

Comment:
I don't think we should add a new field to schema about this, let it be as tableProperty only.